src/cmd/flux-account-service.py

```diff
@@ -0,0 +1,709 @@
+    params = {
+        "path": msg.payload["path"],
+        "bank": msg.payload["bank"],
+    }
+
+    return params
+
+
+def unpack_edit_bank_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "bank": msg.payload["bank"],
+        "shares": msg.payload["shares"],
+        "parent_bank": msg.payload["parent_bank"],
+    }
+
+    return params
+
+
+def unpack_view_job_records_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "output_file": msg.payload["output_file"],
+        "jobid": msg.payload["jobid"],
+        "user": msg.payload["user"],
+        "before_end_time": msg.payload["before_end_time"],
+        "after_start_time": msg.payload["after_start_time"],
+    }
+
+    return params
+
+
+def unpack_update_usage_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "job_archive_db_path": msg.payload["job_archive_db_path"],
+        "priority_decay_half_life": msg.payload["priority_decay_half_life"],
+    }
+
+    return params
+
+
+def unpack_add_queue_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "queue": msg.payload["queue"],
+        "min_nodes_per_job": msg.payload["min_nodes_per_job"],
+        "max_nodes_per_job": msg.payload["max_nodes_per_job"],
+        "max_time_per_job": msg.payload["max_time_per_job"],
+        "priority": msg.payload["priority"],
+    }
+
+    return params
+
+
+def unpack_view_queue_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "queue": msg.payload["queue"],
+    }
+
+    return params
+
+
+def unpack_delete_queue_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "queue": msg.payload["queue"],
+    }
+
+    return params
+
+
+def unpack_edit_queue_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "queue": msg.payload["queue"],
+        "min_nodes_per_job": msg.payload["min_nodes_per_job"],
+        "max_nodes_per_job": msg.payload["max_nodes_per_job"],
+        "max_time_per_job": msg.payload["max_time_per_job"],
+        "priority": msg.payload["priority"],
+    }
+
+    return params
+
+
+def unpack_add_project_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "project": msg.payload["project"],
+    }
+
+    return params
+
+
+def unpack_view_project_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "project": msg.payload["project"],
+    }
+
+    return params
+
+
+def unpack_delete_project_request(msg):
+    params = {
+        "path": msg.payload["path"],
+        "project": msg.payload["project"],
+    }
+
+    return params
+
+
+class Accounting:
+    def __init__(self, flux_handle, conn):
+
+        self.handle = flux_handle
+        self.conn = conn
+
+        # register service with broker. Allows this
+        self.handle.service_register("accounting").get()
+        print("registered accounting service", file=sys.stderr)
```

Comment:
Error message is better, but still a bit confusing:
```
ERROR:root:flux-accounting service is already registered
```
I don't know where the `root` is coming from?

Reply:
Ah, agreed. The error message formatting is coming from the `logging` module, which is what I am using to emit the error. I _think_ by default, the module prints out the name of the logger, which in this case is `root`. But I can format the message to just not include the name, and instead only print out the type of error and the message, so that it will only output:

```console
ERROR:flux-accounting service is already registered
```

I can push that fix up right now.